tsl/test/expected/gapfill.out

```diff
@@ -2029,7 +2029,7 @@ SELECT
  time_bucket_gapfill('1d'::interval,t)
 FROM (VALUES (now()),(now())) v(t)
 WHERE
- t >= now() AND t < now() - '1d'::interval
+ t >= now() AND t < now() - '2d'::interval
```

Comment:
Nit: isn't it safer to just use an interval which is not affected by timezone stuff? Like `24h`?

Reply:
ok switched it all to 1h